Defective Docker Issue

Getting this message when trying to build the docker file:

Successfully built df2db614b282
Successfully tagged ibmosquito/web-hello-c:1.0.0
docker rm -f web-hello-c 2>/dev/null || :
docker run -it --name web-hello-c -p 8000:8000 --volume `pwd`:/outside ibmosquito/web-hello-c:1.0.0 /bin/bash
docker: Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/bin/bash": stat /bin/bash: no such file or directory: unknown.
ERRO[0000] error waiting for container: context canceled
make: *** [Makefile:14: dev] Error 127

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM ubuntu:22.10
 
 RUN apt update && apt install -y build-essential
 

--- a/Makefile
+++ b/Makefile
@@ -8,10 +8,10 @@ PORT:=8000
 all: build run
 
 build:
-	docker build -t $(DOCKER_HUB_ID)/$(NAME):$(VERSION) .
+	docker build --platform linux/amd64 -t $(DOCKER_HUB_ID)/$(NAME):$(VERSION) .
 
 dev: build stop
-	docker run -d --name $(NAME) -p $(PORT):$(PORT) --volume `pwd`:/outside $(DOCKER_HUB_ID)/$(NAME):$(VERSION)
+	docker run -d --name $(NAME) -p $(PORT):$(PORT) $(DOCKER_HUB_ID)/$(NAME):$(VERSION)
 	docker logs -f $(NAME)
 
 run: stop
@@ -19,9 +19,6 @@ run: stop
 
 test:
 	curl -sS localhost:$(PORT)/
-
-exec:
-	docker exec -it $(NAME) /bin/bash
 
 push:
 	docker push $(DOCKER_HUB_ID)/$(NAME):$(VERSION)
@@ -31,7 +28,6 @@ stop:
 
 clean: stop
 	-docker rmi $(DOCKER_HUB_ID)/$(NAME):$(VERSION) 2>/dev/null || :
-	
 
 publish-service:
 	@ARCH=$(ARCH) \
@@ -46,11 +42,11 @@ publish-pattern:
         SERVICE_VERSION="$(SERVICE_VERSION)"\
         PATTERN_NAME="$(PATTERN_NAME)" \
 	hzn exchange pattern publish -f pattern.json
-	
+
 register-pattern:
 	@hzn register --pattern "${HZN_ORG_ID}/$(PATTERN_NAME)"
 
 agent-stop:
 	@hzn unregister -f
 
-.PHONY: all build dev run test exec push stop clean publish-service publish-pattern register-pattern agent-stop
+.PHONY: all build dev run test push stop clean publish-service publish-pattern register-pattern agent-stop

